apps/design-land/src/app/loading-icon/loading-icon.module.ts

```diff
@@ -23,27 +14,10 @@ import { LoadingIconComponent } from './loading-icon.component';
   imports: [
     CommonModule,
     DaffArticleModule,
-    DaffLoadingIconModule,
     DesignLandExampleViewerModule,
     DesignLandLoadingIconRoutingModule,
   ],
 })
 export class LoadingIconModule {
```

Comment:
Can we update this to `DesignLandLoadingIconModule` and do it in the component too since you started doing that to other components in the PR?

Reply:
fixed.